include/ensmallen_bits/sdp/primal_dual_impl.hpp

```diff
@@ -279,9 +202,8 @@ typename MatType::elem_type PrimalDualSolver<DeprecatedSDPType>::Optimize(
   return Optimize(sdp, coordinates, ySparse, yDense, z, callbacks...);
 }
 
-template<typename DeprecatedSDPType>
 template<typename SDPType, typename MatType, typename... CallbackTypes>
-typename MatType::elem_type PrimalDualSolver<DeprecatedSDPType>::Optimize(
+typename MatType::elem_type PrimalDualSolver::Optimize(
     const SDPType& sdp,
```

Comment:
I think we can put this in the line above as well

Reply:
Put what in the line above?  The `const SDPType& sdp,`?

Reply:
Yeah, like:
`typename MatType::elem_type PrimalDualSolver::Optimize(const SDPType& sdp,`
I believe that is less than 80 characters